cider/type/plan/ConstantExpr.cpp

```diff
@@ -34,7 +34,8 @@ JITExprValue& Constant::codegen(JITFunction& func) {
       CIDER_THROW(CiderCompileException,
                   "NULL type literals are not currently supported in this context.");
     case kBOOLEAN:
-      return set_expr_value(func.createConstant(getJITTag(type), get_constval().boolval));
+      return set_expr_value(nullptr,
```

Comment:
`nullptr` should be `func.createConstant(getJITTag(type), false)` to indicate it's a not null constant.

Reply:
It's ok to create a false constant, but then we will need to check null in IR. any suggestions? @bigPYJ1151

Reply:
I think the nullability of Constant should always be false except Constant with kNULLT type. So we could pass a nullptr in most cases.

Reply:
If we set the null to `nullptr`, it probably cause problem when trying to use `expr.getNull()` when we don't know its a constant or not.

Reply:
I add an API `isNullable()`, we should always check it